conf.py

```diff
@@ -59,7 +59,12 @@
 
 html_context = {
     "versions": versions_dict,
-    "current_version": version
+    "current_version": version,
+    "display_github": True, # Integrate GitHub
+    "github_user": "conan-io", # Username
+    "github_repo": "docs", # Repo name
+    "github_version": "master", # Version
```

Comment:
Maybe we should put "develop" here as @lasote thoughts on following the gitflow

Reply:
No, because most edits done in the Github button, are to the master/production branch.